### PR TITLE
ENTMQBR-2403- Added TLS configuration via keystore and truststore

### PR DESCRIPTION
**Pull request by redhatHameed**
Signed-off-by: Abdul Hameed <ahameed@redhat.com>

### CODE REVIEW

File: deploy/catalog_resources/redhat/catalog-source.yaml [hunk 1]
```diff
@@ -244,6 +244,27 @@ items:
                     properties:
                       image:
                         type: string
+                        sslEnabled:
+                          type: boolean
```
Comment (bmozaffa) [inline deploy/catalog_resources/redhat/catalog-source.yaml]: The indentation looks wrong here

File: deploy/crds/broker_v1alpha1_activemqartemis_crd.yaml [hunk 2]
```diff
@@ -29,6 +29,27 @@ spec:
           properties:
             image:
               type: string
+              sslEnabled:
```
Comment (bmozaffa) [inline deploy/crds/broker_v1alpha1_activemqartemis_crd.yaml]: Same here, indent problem

Comment (redhatHameed) [reply deploy/crds/broker_v1alpha1_activemqartemis_crd.yaml]: Fixed

File: deploy/catalog_resources/redhat/catalog-source.yaml (hunk 1 above)
Comment (redhatHameed) [reply deploy/catalog_resources/redhat/catalog-source.yaml]: Fixed

**Comment by RoddieKieley** [comment]
Updated sslEnabled to false, along with minor doc changes. Merged via commit [0e7141847033929921520f2ed6a86603d58f7002](https://github.com/rh-messaging/activemq-artemis-operator/commit/0e7141847033929921520f2ed6a86603d58f7002)

**Pull request closed without merging**